openwisp_controller/config/tests/test_controller.py

```diff
@@ -1090,6 +1091,7 @@ def test_register_403_disabled_registration_setting(self):
         ).count()
         self.assertEqual(count, 0)
 
+    @patch.object(app_settings, 'SHARED_MANAGEMENT_IP_ADDRESS_SPACE', False)
```

Comment:
Will this change classify as a backward incompatible?

As earlier, the default behaviour was to allow IP conflict if devices belonged to different organizations. And now, the default behaviour is avoid such conflict.

Reply:
Yes, I think we can flag it as backward incompatible because if someone relies on this behavior they have to change this setting in order to maintain the old behavior.